spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/ContentOverrides.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.bootstrap.data.builder;
+
+import org.candlepin.dto.api.client.v1.ContentOverrideDTO;
+import org.candlepin.spec.bootstrap.data.util.StringUtil;
+
+
+/**
+ * Class providing factory functions for ContentOverrideDTO instances.
+ */
+public class ContentOverrides {
+
+    private ContentOverrides() {
+        throw new UnsupportedOperationException();
+    }
+
+    public static ContentOverrideDTO random() {
+        return new ContentOverrideDTO()
+            .name(StringUtil.random("name").toLowerCase())
+            .value(StringUtil.random("value").toLowerCase())
+            .contentLabel(StringUtil.random("contentlabel").toLowerCase());
```

Comment:
Is there a reason for the `toLowerCase()` on all of the random string vales?

Reply:
this could be only with name. Because if we calling addActivationKeyContentOverrides(...) and setting name it will parse name into lowerCase.

https://github.com/candlepin/candlepin/blob/3fa2a1062a383476a0961aee2b8bdf88433a637d/src/main/java/org/candlepin/model/ContentOverride.java#L100-L102

Reply:
so I add that we could compare ContentOverrides created by that method and registered to ActivationKeys